pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -93,6 +88,4 @@ stdenv.mkDerivation {
 
     runHook postInstall
   '';
-
-  inherit (unwrapped) meta;
```

Comment:
Check https://github.com/jokogr/nixpkgs/blob/5e0f9dacf090191adfd7a1cea6f13adbf6c66f71/pkgs/applications/display-managers/sddm/default.nix#L70

It's already inherited, correctly?